src/components/Data/Collections/Create.vue

```diff
@@ -22,31 +22,32 @@ export default {
     PageNotAllowed
   },
   props: {
-    index: String
+    indexName: String
   },
   computed: {
     ...mapGetters('auth', ['canCreateCollection']),
     hasRights() {
-      return this.canCreateCollection(this.index, this.collection)
+      return this.canCreateCollection(this.index.name)
+    },
+    index() {
+      return this.$store.direct.getters.index.getOneIndex(this.indexName)
     }
   },
   methods: {
     async create(payload) {
       try {
-        await this.$store.direct.dispatch.index
-          .createCollectionInIndex({
-            index: this.index,
-            collection: payload.name,
-            isRealtimeOnly: payload.realtimeOnly,
-            mapping: payload.mapping,
-            dynamic: payload.dynamic
-          })
-          .then(() => {
-            this.$router.push({
-              name: 'Collections',
-              params: { index: this.index }
-            })
-          })
+        await this.$store.direct.dispatch.index.createCollection({
```

Comment:
Same here, you can use `mapActions` to un-bloat this long call (I never use it bc I just discovered it...) https://vuex.vuejs.org/api/#mapactions